php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -0,0 +1,1100 @@
+      InternalSyntaxToken openCurlyBrace, Optional<List<StatementTree>> statements, InternalSyntaxToken closeCurlyBrace
+  ) {
+    return new NamespaceStatementTreeImpl(
+        namespaceToken,
+        namespaceName.orNull(),
+        openCurlyBrace,
+        optionalList(statements),
+        closeCurlyBrace
+    );
+  }
+
+  public InlineHTMLTree inlineHTML(InternalSyntaxToken inlineHTMLToken) {
+    return new InlineHTMLTreeImpl(inlineHTMLToken);
+  }
+
+  public DeclareStatementTree shortDeclareStatement(DeclareStatementHead declareStatementHead, InternalSyntaxToken eosToken) {
+    return new DeclareStatementTreeImpl(declareStatementHead, eosToken);
+  }
+
+  public DeclareStatementHead declareStatementHead(
+      InternalSyntaxToken declareToken, InternalSyntaxToken openParenthesisToken,
+      VariableDeclarationTree firstDirective, Optional<List<Tuple<InternalSyntaxToken, VariableDeclarationTree>>> optionalDirectives,
+      InternalSyntaxToken closeParenthesisToken
+  ) {
+    return new DeclareStatementHead(
+        declareToken,
+        openParenthesisToken,
+        separatedList(firstDirective, optionalDirectives),
+        closeParenthesisToken
+    );
+  }
+
+  public DeclareStatementTree declareStatementWithOneStatement(DeclareStatementHead declareStatementHead, StatementTree statement) {
+    return new DeclareStatementTreeImpl(declareStatementHead, statement);
+  }
+
+  public DeclareStatementTree alternativeDeclareStatement(
+      DeclareStatementHead declareStatementHead, InternalSyntaxToken colonToken,
+      Optional<List<StatementTree>> statements,
+      InternalSyntaxToken enddeclareToken, InternalSyntaxToken eosToken
+  ) {
+    return new DeclareStatementTreeImpl(declareStatementHead, colonToken, optionalList(statements), enddeclareToken, eosToken);
+  }
+
+  public StaticStatementTree staticStatement(InternalSyntaxToken staticToken, VariableDeclarationTree variable, Optional<List<Tuple<InternalSyntaxToken, VariableDeclarationTree>>> listOptional, InternalSyntaxToken eosToken) {
+    return new StaticStatementTreeImpl(staticToken, separatedList(variable, listOptional), eosToken);
+  }
+
+  /**
+   * [ END ] Statement
+   */
+
+  public ExpressionTree expression(InternalSyntaxToken token) {
+    return new VariableIdentifierTreeImpl(new IdentifierTreeImpl(token));
+  }
+
+  /**
+   * [ START ] Expression
+   */
+
+  public LiteralTree numericLiteral(InternalSyntaxToken token) {
+    return new LiteralTreeImpl(Tree.Kind.NUMERIC_LITERAL, token);
+  }
+
+  public LiteralTree regularStringLiteral(InternalSyntaxToken token) {
+    return new LiteralTreeImpl(Tree.Kind.REGULAR_STRING_LITERAL, token);
+  }
+
+  public LiteralTree booleanLiteral(InternalSyntaxToken token) {
+    return new LiteralTreeImpl(Tree.Kind.BOOLEAN_LITERAL, token);
+  }
+
+  public LiteralTree nullLiteral(InternalSyntaxToken token) {
+   return new LiteralTreeImpl(Tree.Kind.NULL_LITERAL, token);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)